packages/ceramic-core/src/ceramic.ts

```diff
@@ -268,8 +270,7 @@ class Ceramic implements CeramicApi {
     }
 
     doc = await Document.create(genesisCid, doctypeHandler, this.dispatcher, this.pinStore, this.context, opts, this._validateDocs);
-    const normalizedId = DoctypeUtils.normalizeDocId(doc.id)
-    this._docmap[normalizedId] = doc
+    this._docmap[doc.id.toString()] = doc
```

Comment:
Yup, here we are storing it using the docid, so the previous comment is correct.